pkg/kubelet/kubelet_node_status.go

```diff
@@ -114,10 +145,10 @@ func (kl *Kubelet) tryRegisterWithAPIServer(node *v1.Node) bool {
 	// Edge case: the node was previously registered; reconcile
 	// the value of the controller-managed attach-detach
 	// annotation.
-	requiresUpdate := kl.reconcileCMADAnnotationWithExistingNode(node, existingNode)
-	requiresUpdate = kl.updateDefaultLabels(node, existingNode) || requiresUpdate
+	requiresUpdate := reconcileCMADAnnotationWithExistingNode(node, existingNode)
+	requiresUpdate = updateDefaultLabels(node, existingNode) || requiresUpdate
 	requiresUpdate = kl.reconcileExtendedResource(node, existingNode) || requiresUpdate
```

Comment:
I wonder what's the benefit of making these method `kubelet`- independent? If there are benefits, why not change this method as well and pass the configuration parameter it needs from this method down to `reconcileExtendedResource`? Making these calls inconsistent like this looks suspicious

Reply:
These don't rely on any kubelet struct, why should we test them before build a kubelet struct?


> why not change this method as well and pass the configuration parameter it needs from this method down to `reconcileExtendedResource`? Making these calls inconsistent like this looks suspicious

I agree with you.

Reply:
updated